service/worker/scheduler/activities.go

```diff
@@ -65,12 +70,19 @@ type (
 	}
 )
 
+const (
+	eventStorageSize = 2 * 1024 * 1024
+	// I do not know the real overhead size, 100 is just a number
```

Comment:
here is a good example of comment go stale. I updated recordOverheadSize to be 1000 and did not update the comment.